test/key-value-model.test.js

```diff
@@ -138,16 +139,16 @@ describe('KeyValueModel', function() {
   });
 
   function setupAppAndCacheItem() {
-    app = loopback({ localRegistry: true, loadBuiltinModels: true });
+    app = loopback({localRegistry: true, loadBuiltinModels: true});
     app.use(loopback.rest());
 
     CacheItem = app.registry.createModel({
       name: 'CacheItem',
       base: 'KeyValueModel',
     });
 
-    app.dataSource('kv', { connector: 'kv-memory' });
-    app.model(CacheItem, { dataSource: 'kv' });
+    app.dataSource('kv', {adapter: 'kv-memory'});
```

Comment:
I don't understand why is this change necessary, could you please explain?

The `connector` option is the canonical way of configuring data-source connector, I am not sure if `adapter` is even documented.